### ISSUE TITLE
Error: Invalid text field on Linter Response

### ISSUE THREAD

**Issue by thibaudgg** [AtomLinter/linter-reek#15]
I got this error with linter-reek 2.0.1 & linter 1.8.0, disabling linter-reek make it goes away.

```
Error: Invalid text field on Linter Response
  at /Users/Thibaud/.atom/packages/linter/lib/validate.coffee:33:19
  at Array.forEach (native)
  at Object.module.exports.Validate.messages (/Users/Thibaud/.atom/packages/linter/lib/validate.coffee:25:14)
  at MessageRegistry.set (/Users/Thibaud/.atom/packages/linter/lib/message-registry.js:61:18)
  at /Users/Thibaud/.atom/packages/linter/lib/linter.coffee:27:17
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at /Users/Thibaud/.atom/packages/linter/lib/linter-registry.coffee:62:18
```

**Comment by steelbrain** [comment]
@dhagerty9009 this linter is providing a non-string content on `text` field.

**Comment by thibaudgg** [comment]
It's still an issue for me on Atom 1.1, too bad.

**Comment by Arcanemagus** [comment]
Do you have an example file that triggers this @thibaudgg?

**Comment by thibaudgg** [comment]
Nothing special, I just save some random ruby files and it happens every time.

**Comment by dhagerty9009** [comment]
It's a known issue. I apologize for leaving it unfixed - work and school plus wedding planning have left me with precious little free time. I'm hoping to get to the issue this weekend. The main thing currently is that I'm performing a regex on the unformatted output from the `reek` command. I'd like to move the output to JSON, an option provided by Reek, which would make parsing errors trivial.

**Comment by dhagerty9009** [comment]
This should be fixed now that I merged in PR #13.

**Comment by thibaudgg** [comment]
Awesome it works now with 2.0.2! @dhagerty9009 thanks :gift:!
